src/coreclr-debug/util.ts

```diff
@@ -0,0 +1,117 @@
+/*---------------------------------------------------------------------------------------------
+ *  Copyright (c) Microsoft Corporation. All rights reserved.
+ *  Licensed under the MIT License. See License.txt in the project root for license information.
+ *--------------------------------------------------------------------------------------------*/
+'use strict';
+
+import * as path from 'path';
+import * as fs from 'fs';
+import * as os from 'os';
+
+let _extensionDir: string = '';
+let _coreClrDebugDir: string = '';
+let _debugAdapterDir: string = '';
+let _installLogPath: string = '';
+let _installBeginFilePath: string = '';
+let _installCompleteFilePath: string = '';
+
+export function setExtensionDir(extensionDir: string) {
```

Comment:
Hmmm... a bunch of state and function that operate on it. Maybe just make a class? :-)